static/js/changelog.controllers.js

```diff
@@ -173,10 +173,9 @@
         .controller('AutorefreshController', function ($scope, $interval, ChangelogApi) {
             var refreshInterval = 10000,
                 stepGranularity = 100,
-                elapsed = 0,
-                intervalId;
+                elapsed = 0;               
             $scope.percent = 0;
-            intervalId = $interval(function () {
```

Comment:
Yes, there is no other occurrence of this variable. Also, the variable is not used in index.html under the controller.